source/overview/security.rst

```diff
@@ -2,74 +2,133 @@
-   - Custom `data retention policies on messages and file uploads <https://docs.mattermost.com/administration/data-retention.html>`_ is available (E20). A daily data deletion job can be scheduled that deletes messages from the database and user interface, and file uploads from local file storage or Amazon S3, which exceed the specified retention period. 
-   - The `output and archives of server logs <https://docs.mattermost.com/administration/config-settings.html#file-log-directory>`_ can be saved to a directory of your choice. Mattermost server logs plus logs from your web proxy can provide an end-to-end history of system usage.
-   - `Ad hoc compliance reports of messaging by user, date range, and keyword, including edited and deleted messages <https://docs.mattermost.com/administration/compliance.html>`_ are available (E20). To protect against unauthorized use, all ad hoc report requests are logged.
-   - Daily compliance reports compatible with 3rd party compliance solutions such as `Global Relay <https://docs.mattermost.com/administration/compliance.html#global-relay-support>`_ are also available (E20).
+Mattermost stores a complete history of messages by default.
+This includes edits and deletes and all uploaded files. 
+User interface actions for *deleting* messages and channels remove the data only from the user interface but not from your database. 
+If compliance guidelines demand more then administrators can turn off users' ability to edit and delete their messages after posting.
+
+Custom `data retention policies on messages and file uploads <https://docs.mattermost.com/administration/data-retention.html>`_ is available in Mattermost Enterprise Edition E20. 
+A daily data deletion job can be scheduled to purge messages from the database and user interfaces and purge file uploads from local file storage or Amazon S3 that have exceeded the specified retention period. 
+
+The `output and archives of server logs <https://docs.mattermost.com/administration/config-settings.html#file-log-directory>`_ can be saved to a directory of an organization's choice. 
+
+Mattermost server logs and logs from a web proxy can provide an end-to-end history of system usage.
+`Ad hoc compliance reports of messaging by user, 
+date range, 
+and keyword, 
+including edited and deleted messages <https://docs.mattermost.com/administration/compliance.html>`_,
+are available on Mattermost Enterprise Edition E20. 
+
+All ad hoc report requests are logged to protect against unauthorized use. 
+Daily compliance reports compatible with third party compliance solutions like `Global Relay <https://docs.mattermost.com/administration/compliance.html#global-relay-support>`_ are also available on Mattermost Enterprise Edition E20.
```

Comment:
Note - Should we take this out until we add the built-in Global Relay support

Reply:
yes, I think so. I will leave this edit to you, as I don't know how it got in. But if it is not yet implemented then it should be in a future features section or something like that.